linters/groovy/.groovylintrc.json

```diff
@@ -8,6 +8,8 @@
 
     "size.ParameterCount": { "maxParameters": 9 },
 
-    "dry.DuplicateStringLiteral": { "ignoreStrings": ",/,Jenkinsfile" }
+    "dry.DuplicateStringLiteral": { "ignoreStrings": ",/,\n,Jenkinsfile" },
+
+    "unnecessary.UnnecessaryGetter": { "checkIsMethods": false, "ignoreMethodNames": "getJenkinsfilePath" }
```

Comment:
probably would rename `getJenkinsfilePath` to something like `findJenkinsfilePath`